crates/sui-indexer-alt-framework/migrations/2024-12-11-234958_cp_mapping/up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS cp_mapping
+(
+    cp                                  BIGINT       PRIMARY KEY,
```

Comment:
nit: in `sui-indexer-alt-schema`, we use `cp_sequence_number`. Do we want to follow the naming here too?